dev-packages/browser-integration-tests/suites/feedback/captureFeedbackAndReplay/hasSampling/init.js

```diff
@@ -5,7 +5,7 @@ window.Sentry = Sentry;
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   replaysOnErrorSampleRate: 1.0,
-  replaysSessionSampleRate: 0,
+  replaysSessionSampleRate: 1.0,
```

Comment:
I think this was intentional to make sure that replay is captured when in buffering mode

Reply:
it is not actually doing this today, I believe? At least the test failed 🤔 I merged this in now so we are unblocked, we can revert/tweak this a bit in a follow up!